### PR TITLE
Document FAF OAuth

### PR DESCRIPTION
**Pull request by IDragonfire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,17 @@ Logs are viewable by
 
 If you want to view the raw JSON on the website, then you will need to allow 'Allow-Control-Allow-Origin'' in the browser.
 Here is an example extension for Chrome - (https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en)
+
+# Using OAuth 2.0
+
+This API implements [OAuth 2.0](http://oauth.net/2/), you find a basic OAuth tutorial [here](https://aaronparecki.com/2012/07/29/2/oauth2-simplified). Here a faf tutorial:
```
Comment (micheljung) [inline README.md]: FAF

File: README.md [hunk 2]
```diff
@@ -55,3 +55,17 @@ Logs are viewable by
...
+
+## Request Authorization Code
+http://tools.ietf.org/html/rfc6749#section-4.1.1
+* Add your OAuth Client into the table `oauth_clients`
```
Comment (micheljung) [inline README.md]: client

File: README.md [hunk 3]
```diff
@@ -55,3 +55,17 @@ Logs are viewable by
...
+* Request it `/oauth/authorize?client_id=<YourClientId>&response_type=code`
+
+## Request Authorization Token
+https://tools.ietf.org/html/rfc6749#section-4.2.1
+* Add your OAuth Client into the table `oauth_clients`
```
Comment (micheljung) [inline README.md]: client

**Comment by IDragonfire** [comment]
@micheljung done

**Pull request merged**